environment/git.go

```diff
@@ -35,6 +35,45 @@ func (env *Environment) GetWorktreePath() (string, error) {
 	return homedir.Expand(fmt.Sprintf("~/.config/container-use/worktrees/%s", env.ID))
 }
 
+func (env *Environment) DeleteWorktree() error {
+	worktreePath, err := env.GetWorktreePath()
+	if err != nil {
+		return err
+	}
+	parentDir := filepath.Dir(worktreePath)
+	fmt.Printf("Deleting parent directory of worktree at %s\n", parentDir)
+	return os.RemoveAll(parentDir)
```

Comment:
this is the only thing surprising to me about this impl, using `rm` instead of `git worktree remove` -- that said it's pretty [clearly documented](https://git-scm.com/docs/git-worktree#_description) that this approach is safe and prune will clean up the admin files in cuRepoPath.